catalyst/contrib/callbacks/draw_masks_callback.py

```diff
@@ -0,0 +1,181 @@
+from typing import Iterable, Optional, TYPE_CHECKING
+import os
+
+import numpy as np
+from skimage.color import label2rgb
+
+import torch
+from torch.utils.tensorboard import SummaryWriter
+
+from catalyst import utils
+from catalyst.callbacks import ILoggerCallback
+from catalyst.contrib.utils.cv.tensor import tensor_to_ndimage
+from catalyst.core.callback import CallbackNode, CallbackOrder
+
+if TYPE_CHECKING:
+    from catalyst.core.runner import IRunner
+
+
+class DrawMasksCallback(ILoggerCallback):
+    """
+    Logger callback draw masks for common segmentation task: image -> masks
+    """
+
+    def __init__(
+        self,
+        pred_mask_key: str,
+        image_key: Optional[str] = None,
+        gt_mask_key: Optional[str] = None,
+        mask2show: Optional[Iterable[int]] = None,
+        activation: Optional[str] = "Sigmoid",
```

Comment:
can't be `None` -> `optional` redundant

Reply:
No, if there isn't image_key, than only the predicted mask will be drawn